lib/ecto/adapters/postgres/connection.ex

```diff
@@ -165,6 +165,15 @@ if Code.ensure_loaded?(Postgrex) do
        values, on_conflict(on_conflict, header) | returning(returning)]
     end
 
+    def parse_log_param({binary, Postgrex.Extensions.UUID}) do
+      case Ecto.UUID.load(binary) do
+        {:ok, uuid} -> uuid
+        :error -> binary
+      end
+    end
+
+    def parse_log_param({param, _type}), do: param
```

Comment:
Can this be added to the callbacks for `Ecto.Adapters.SQL.Connection`

Reply:
Good call, but it's an optional callback so I'll have to mark it as such.